### ISSUE TITLE
Clarification - how the plugin works on multiple cards

### ISSUE THREAD

**Issue by chezfo** [aleksandar-j/set-added-date#3]
Thank you for the addition!!

I wanted to find out how the add-on works, if I select let's say 30 cards in the browser, and set the add-on on them, will each one get a time after the card before it?

I mean I mean to ask if he will receive an addition date in the order it appears in the browser?

**Comment by aleksandar-j** [comment]
Yes, card and note added date must be unique so each one will get a different date. Differences will be, for most purposes, imperceptible (each subsequent date +1 millisecond). If you were to select 10000 cards you'd notice about 10 second difference between cards with lowest and highest date.
Order is not kept. After setting new added date those 30 cards may shuffle around in the browser. This can be changed, though I am unsure how useful it would be?

**Comment by chezfo** [comment]
Why is the order not preserved, because it does not divide the dates out of the order they are in the browser?

The matter is important to me, for the option to sort decks according to the order of their addition, (in filtered decks), and I would like to determine the order according to other parameters that do not exist today, such as alphabetical order.

**Comment by aleksandar-j** [comment]
As an example, assume you wish to select three cards (A,B,C) with creation dates:
- A: "2020-01-01"
- B: "2021-01-01"
- C: "2022-01-01"

Then you set new creation date to "2023-01-01 00:00:00".
Current version of addon could, for example, set:
- B: "2023-01-01 00:00:00 + 0 milliseconds"
- C: "2023-01-01 00:00:00 + 1 milliseconds"
- A: "2023-01-01 00:00:00 + 2 milliseconds"

Therefore, order is not kept (A<B<C ⇒ B<C<A).

I've updated the addon logic on my end so this no longer happens; meaning cards now keep their relative order after edits (in example above A must come before B which must come before C). I'll update the code here and on AnkiWeb after further testing.

If that's not what you had in mind please provide an example.

**Comment by chezfo** [comment]
According to the new definition, the cards can be arranged in the browser according to any parameter, and the plugin will determine the date according to the order that appears in the browser when running the command?

If so, that's really great :)
